include/image.mk

```diff
@@ -589,12 +604,16 @@ endef
 
 define Device/DumpInfo
 Target-Profile: DEVICE_$(1)
-Target-Profile-Name: $(DEVICE_TITLE)
+Target-Profile-Name: $(DEVICE_TITLE)$(if $(strip $(DEVICE_ALT0_TITLE)), ($(DEVICE_ALT0_TITLE)))
 Target-Profile-Packages: $(DEVICE_PACKAGES)
 Target-Profile-hasImageMetadata: $(if $(foreach image,$(IMAGES),$(findstring append-metadata,$(IMAGE/$(image)))),1,0)
 Target-Profile-SupportedDevices: $(SUPPORTED_DEVICES)
 Target-Profile-Description:
 $(DEVICE_DESCRIPTION)
+$(if $(strip $(DEVICE_ALT0_TITLE)),Alternative device titels:)
+$(DEVICE_ALT0_TITLE)
+$(DEVICE_ALT1_TITLE)
```

Comment:
This will create empty lines if ALT1 etc. is not set. Maybe use an if for each line?

Reply:
Just tested it, and it does NOT create additional empty lines. So this seems to be healed somewhere else ...

Reply:
Nice, good to know that deep-Make is taking care of that somewhere...